src/main/java/teammates/ui/controller/InstructorFeedbackAbstractAction.java

```diff
@@ -30,10 +30,9 @@
      * @param isCreatingNewSession true if creating a new session; false if editing an existing session.
      * @return feedback session attributes object.
      */
-    protected FeedbackSessionAttributes extractFeedbackSessionData(boolean isCreatingNewSession) {
-        // TODO make this method stateless
-
-        // Null checks for parameters not done as null values do not affect data integrity
+    protected FeedbackSessionAttributes extractFeedbackSessionData(boolean isCreatingNewSession) {    
```

Comment:
You can set up Eclipse to auto remove all trailing spaces on save in editor, save actions.